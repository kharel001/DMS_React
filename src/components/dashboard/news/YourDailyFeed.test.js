import React from 'react';
import YourDailyFeed from './YourDailyFeed';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const setup = (props = {}) => {
    return shallow( < YourDailyFeed {...props}  />
    )
}

const findByTestAttr=(wrapper, val)=>{
    return wrapper.find(`[data-test='${val}']`)
}


const dummy_props={
    data:{
        name:"name1", 
        desc:"Description", 
        image:"image1"
    }
}
describe("YourDailyFeed Comp",()=>{
it("Snapshot",()=>{
    const wrapper= setup(dummy_props);
    expect(wrapper).toMatchSnapshot()

})
it("Props YourDailyFeed", () => {
    const wrapper= setup(dummy_props);   
    const project= findByTestAttr(wrapper, "feedCellComp") 
    expect(project).toHaveLength(5);
    // expect(project.props().data).toEqual(dummy_data)

})

})