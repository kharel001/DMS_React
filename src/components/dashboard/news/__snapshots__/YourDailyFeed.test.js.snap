// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YourDailyFeed Comp Snapshot 1`] = `ShallowWrapper {}`;
